test/transcoding/SPV_INTEL_media_block_io.cl

```diff
@@ -0,0 +1,214 @@
+// RUN: %clang_cc1 -triple spir-unknown-unknown -O1 -cl-std=CL2.0 -finclude-default-header -emit-llvm-bc %s -o %t.bc
+// RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_media_block_io -spirv-text -o - | FileCheck %s --check-prefix=CHECK-SPIRV
```

Comment:
I suggest generate text representation from the binary one - in this case, you don't need to pass `--spirv-ext` option since the translator doesn't check for allowed/disallowed extensions while translating to/from text representation.

```
// RUN: llvm-spirv %t.spv -to-text -o - | FileCheck %s --check-prefix=CHECK-SPIRV
```

Reply:
Done